.github/workflows/helmfile_staging_apply.yaml

```diff
@@ -0,0 +1,44 @@
+name: Staging - Helmfile Apply 
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  HELMFILE_FILE_PATH: ${{ github.workspace }}/helmfile
+  STAGING_AWS_ACCOUNT: ${{ secrets.STAGING_AWS_ACCOUNT }}
+
+jobs:
+  helmfile-diff:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Inject token authentication
+        run: |
+          git config --global url."https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/".insteadOf "https://github.com/"
```

Comment:
Minor comment here, if you wanted to move away from a hard-coded token, you could replace this with an SRE GitHub app's temp token:
https://github.com/cds-snc/gcds-components/blob/424b138d6ebf8e3df77b2e265ff386bf1e39d7cf/.github/workflows/release-generator.yml#L15-L19

Reply:
Nice, not opposed to this but I think it's out of scope for this particular PR - I'll create a card to migrate all workflows to this method.

Reply:
Card created:
https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/292